Update User-Agent Client Hints docs (#18928)
mdn/content: files/en-us/web/api/navigatoruadata/gethighentropyvalues/index.md

A {{jsxref("Promise")}} that resolves to an object containing some or all of the following values (based on the hints requested):

- `brands`
  - : Returns an array of objects containing `brand` and `version` specifying the browser brand and its version (the same information as provided by {{domxref("NavigatorUAData.brands")}}).
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
- `mobile`

>   - : Returns `true` if the user-agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).

Reviewer: ```suggestion
  - : Returns `true` if the user agent is running on a mobile device (the same information as provided by {{domxref("NavigatorUAData.mobile")}}).
```

      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Mobile")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
- `platform`

>   - : Returns a string describing the platform the user-agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).

Reviewer: ```suggestion
  - : Returns a string describing the platform the user agent is running on, like `"Windows"` (the same information as provided by {{domxref("NavigatorUAData.platform")}}).
```

      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform")}} header (a [low-entropy client hint](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)).
- `architecture`
  - : A string containing the platform architecture. For example, `"x86"`.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Arch")}} header after the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

- `bitness`
  - : A string containing the architecture bitness. For example, `"32"` or `"64"`.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Bitness")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

- `model`
  - : A string containing the model of mobile device. For example, `"Pixel 2XL"`. If device is not a mobile device or if device model is not known, `model` will be `""`.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Model")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

- `platformVersion`
  - : A string containing the platform version. Platform name itself is always available as low-entropy hint `platform`. For example, `"10.0"`.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform-Version")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Platform-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

- `uaFullVersion` {{deprecated_inline}}
  - : A string containing the full browser version. For example, `"103.0.5060.134"`. Deprecated in favor of `fullVersionList`.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

- `fullVersionList`
  - : An array of objects with properties `"brand"` and `"version"` representing the browser name and full version respectively.
      For example, `{"brand": "Google Chrome", "version": "103.0.5060.134"}, {"brand": "Chromium", "version": "103.0.5060.134"}`.
      Please note that one object may intentionally contain invalid information to prevent sites from relying on a fixed list of browsers.

>       Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version-List")}} header if server explicitly requests it in {{HTTPHeader("Accept-CH")}}.

Reviewer: ```suggestion
      Note that this information can be sent to a server in the {{HTTPHeader("Sec-CH-UA-Full-Version-List")}} header if the server explicitly requests it in the {{HTTPHeader("Accept-CH")}} header.
```

## See also

- These values are also available as via HTTP request headers:
  - [Low-entropy client hints](/en-US/docs/Web/HTTP/Client_hints#low_entropy_hints)) are sent automatically:
    - {{HTTPHeader("Sec-CH-UA")}}
    - {{HTTPHeader("Sec-CH-UA-Mobile")}}
    - {{HTTPHeader("Sec-CH-UA-Platform")}}

>   - Server can request to receive high-entropy client hints on subsequent requests via {{HTTPHeader("Accept-CH")}}:

Reviewer: ```suggestion
  - Servers can request to receive high-entropy client hints on subsequent requests, using the {{HTTPHeader("Accept-CH")}} header:
```
